apicompat/netstandard1x/netstandard1x.depproj

```diff
@@ -1,14 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)'==''">.NETStandard,Version=v1.6</NuGetTargetMoniker>
```

Comment:
As this project is now sdk based I don't think we need the NuGetTargetMonikers anymore in https://github.com/dotnet/standard/blob/master/apicompat/netstandard1x/netstandard.builds#L30.

Reply:
Good call, I'll remove that